fix "error: unrecognized command line option '-fuse-ld=--enable-gold=default'"
see also https://github.com/bazelbuild/bazel/issues/9392
--- tools/cpp/unix_cc_configure.bzl.orig	2019-09-26 20:39:13.350751339 +0200
+++ tools/cpp/unix_cc_configure.bzl	2019-09-26 20:39:44.033325970 +0200
@@ -200,6 +200,8 @@
         for flag in line.split(" "):
             if flag.find("gold") == -1:
                 continue
+            if flag.find("enable-gold") > -1 or flag.find("with-plugin-ld") > -1:
+                continue

             # flag is '-fuse-ld=gold' for GCC or "/usr/lib/ld.gold" for Clang
             # strip space, single quote, and double quotes
